client/app/scripts/components/search.js

```diff
@@ -130,14 +133,17 @@ class Search extends React.Component {
             </label>
           </div>
           {!showPinnedSearches && <div className="search-hint">
-            {getHint(nodes)}
+            {getHint(nodes)} <span className="search-help-link" onClick={onClickHelp}>
```

Comment:
HELP! is hard to read. Why not use a question mark in a circle?

Also, the search bar collapses again when help is clicked, which was a bit confusing.